### PR TITLE
[13.0][MIG] hr_expense_cancel

### PR DESCRIPTION
**Pull request by Saran440**
include IMP from this pr  https://github.com/OCA/hr/pull/739

### CODE REVIEW

**Comment by manuelcalerosolis** [comment]
I have searched the PR in the "hr" section not in "hr_expense" for that reason I have not found it. Yes, I will review it.

File: hr_expense_cancel/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Hr expense cancel",
+    "version": "13.0.1.0.0",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/hr",
```
Comment (manuelcalerosolis) [inline hr_expense_cancel/__manifest__.py]: Please change to :

 "website": "https://github.com/OCA/hr-expense",

File: hr_expense_cancel/README.rst [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fhr-lightgray.png?logo=github
+    :target: https://github.com/OCA/hr/tree/13.0/hr_expense_cancel
```
Comment (manuelcalerosolis) [inline hr_expense_cancel/README.rst]: Please change :

:target: https://github.com/OCA/hr-expense/tree/13.0/hr_expense_cancel

File: hr_expense_cancel/README.rst [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    :alt: OCA/hr
```
Comment (manuelcalerosolis) [inline hr_expense_cancel/README.rst]: Please change :

OCA/hr-expense

File: hr_expense_cancel/README.rst [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-13-0/hr-13-0-hr_expense_cancel
```
Comment (manuelcalerosolis) [inline hr_expense_cancel/README.rst]: Please change :

    :target: https://translation.odoo-community.org/projects/hr-expense-13-0/hr-expense-13-0-hr_expense_cancel

File: hr_expense_cancel/README.rst [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+#. After that, the *Expense report* will have an associated journal entry
+   reconciled with a payment
+#. Click on *Cancel* button
+#. The *Expense report* will be set to *Submitted* state; the journal entry and
+   the payment will be deleted
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/hr/issues>`_.
```
Comment (manuelcalerosolis) [inline hr_expense_cancel/README.rst]: Please change:

Bugs are tracked on `GitHub Issues <https://github.com/OCA/hr-expense/issues>`_.

File: hr_expense_cancel/README.rst [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/hr/issues/new?body=module:%20hr_expense_cancel%0Aversion:%2013.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (manuelcalerosolis) [inline hr_expense_cancel/README.rst]: Please change:

`feedback <https://github.com/OCA/hr-expense/issues/new?body=module:%20hr_expense_cancel%0Aversion:%2013.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.

File: hr_expense_cancel/README.rst [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/hr <https://github.com/OCA/hr/tree/13.0/hr_expense_cancel>`_ project on GitHub.
```
Comment (manuelcalerosolis) [inline hr_expense_cancel/README.rst]: Please change :

This module is part of the `OCA/hr-expense <https://github.com/OCA/hr-expense/tree/13.0/hr_expense_cancel>`_ project on GitHub.

**Review by manuelcalerosolis — CHANGES_REQUESTED** [review]

**Comment by Saran440** [comment]
@manuelcalerosolis Fixed. Thank you for your review.

**Review by ps-tubtim — APPROVED** [review]

File: hr_expense_cancel/wizard/hr_expense_sheet_register_payment.py [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+from odoo import models
+
+
+class HrExpenseSheetRegisterPaymentWizard(models.TransientModel):
+    _inherit = "hr.expense.sheet.register.payment.wizard"
+
+    def _get_payment_vals(self):
+        sup = super(HrExpenseSheetRegisterPaymentWizard, self)
+        res = sup._get_payment_vals()
```
Comment (i-vyshnevska) [inline hr_expense_cancel/wizard/hr_expense_sheet_register_payment.py]:
```suggestion
        res = super()._get_payment_vals()
```

**Comment by Saran440** [comment]
@i-vyshnevska Thank you for your reviewed. Please you re-check again.
If it's okay, please approve this pr :)

**Review by i-vyshnevska — APPROVED** [review]

File: setup/.setuptools-odoo-make-default-ignore [hunk 9]
```diff
@@ -0,0 +1,2 @@
+# addons listed in this file are ignored by
```
Comment (i-vyshnevska) [inline setup/.setuptools-odoo-make-default-ignore:1]: just FYI setup files should be generated by bot, there is no need to add them

**Review by i-vyshnevska — APPROVED** [review]

File: setup/.setuptools-odoo-make-default-ignore (hunk 9 above)
Comment (Saran440) [reply setup/.setuptools-odoo-make-default-ignore:1]: this file generated by pre-commit. if i don't add it, it will not travis green.

**Review by ivantodorovich — APPROVED** [review]
LGTM, functional test :+1:

**Comment by kittiu** [comment]
@OCA/human-resources-maintainers can this merged?

**Pull request merged**